src/boot_O2_g3/idle.c

```diff
@@ -110,10 +115,11 @@ void Idle_InitVideo(void) {
 
 void Idle_ThreadEntry(void* arg) {
     Idle_InitVideo();
-    osCreatePiManager(150, &gPiMgrCmdQ, sPiMgrCmdBuff, ARRAY_COUNT(sPiMgrCmdBuff));
+    osCreatePiManager(OS_PRIORITY_PIMGR, &gPiMgrCmdQueue, sPiMgrCmdBuff, ARRAY_COUNT(sPiMgrCmdBuff));
     StackCheck_Init(&sMainStackInfo, sMainStack, STACK_TOP(sMainStack), 0, 0x400, "main");
     osCreateThread(&sMainThread, Z_THREAD_ID_MAIN, Main_ThreadEntry, arg, STACK_TOP(sMainStack), Z_PRIORITY_MAIN);
     osStartThread(&sMainThread);
-    osSetThreadPri(NULL, 0);
+    osSetThreadPri(NULL, OS_PRIORITY_IDLE);
 
-    do { } while (true); }
+    for (;;) {}
```

Comment:
```suggestion
    while (true) {}
```
I prefer `while (true)` for infinite loops, thoughts?

Reply:
I also prefer that, however IDO does not. The `while (true)` actually generates a few additional `nops`, so doesn't match. Using `for (;;)` over the `do while(true)' also prevents the weird brace formatting, which is why I changed it.